stac_fastapi/types/stac_fastapi/types/core.py

```diff
@@ -124,9 +124,22 @@ class BaseCoreClient(abc.ABC):
     description: str = attr.ib(default="stac-fastapi")
     extensions: List[ApiExtension] = attr.ib(default=attr.Factory(list))
 
-    def extension_is_enabled(self, extension: Type[ApiExtension]) -> bool:
+    def extension_is_enabled(self, extension: str) -> bool:
         """Check if an api extension is enabled."""
-        return any([isinstance(ext, extension) for ext in self.extensions])
+        return any([type(ext).__name__ == extension for ext in self.extensions])
+
+    def list_conformance_classes(self):
+        """Return a list of conformance classes, including implemented extensions."""
+        base_conformance = [
```

Comment:
I believe this is the set of conformance classes we want to use https://github.com/stac-utils/stac-fastapi/issues/159

Reply:
yes, these are the old conformance classes here. #159 has the most up to date ones.

Reply:
presumably we are on beta2 now. 

e.g. https://api.stacspec.org/v1.0.0-beta.2/core

Offically 1.0.0 has been released but it looks like the openapi schemas are not there yet.

Reply:
Correct.

Reply:
Recent updates to the library have pushed things from 1.0.0-beta.2 to 1.0.0 in case that was missed